pulp_riscv_dbg.core

```diff
@@ -21,22 +20,25 @@ filesets:
       - vendor/pulp_riscv_dbg/src/dmi_jtag.sv
     file_type: systemVerilogSource
 
-  files_vivado:
+  files_xilinx_bscane_tap:
```

Comment:
We don't want the bscane tap in Sonata at all so we should remove this.

Reply:
This has now been removed: https://github.com/lowRISC/sonata-system/pull/56